plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -222,6 +222,13 @@ public void testCharVarcharComparison()
                 .hasMessage("Unsupported type: char(3)");
     }
 
+    @Test
+    @Override
+    public void testSelectInformationSchemaColumns()
+    {
+        executeExclusively(super::testSelectInformationSchemaColumns);
```

Comment:
would be good to have a code comment explaining why

Reply:
https://github.com/trinodb/trino/pull/21670